test/unit/js/pauseResume.test.js

```diff
@@ -10,13 +10,46 @@ QUnit.config.reorder = false;
 var hooks = {
 	beforeEach: function() {
 		this.$el = $("#box1");
+		this.$el.each(function() {
```

Comment:
it can be silly question, but $el is referring an element which has id. Assuming that, $el.each doesn't make sense, considering $el is an element not array.